client/src/views/Organizations/OrganizationPage.tsx

```diff
@@ -47,7 +52,23 @@ const OrganizationPage = () => {
     () => getEvents(+organizationId!)
   );
 
-  console.log(eventsData);
+  // Determine the currently active team, if one hasn't been chosen, use the organization-wide team
```

Comment:
How does this work? When I refresh the page on my computer I loose state for an id and get an error page. If I comment this whole use effect hook out then the page will refresh properly. Do you have a different DB or something set up that might cause this error on my computer?

Reply:
The changes to the DB structure means it needs to be reseeded, and then that will work because it's currently looking for an auto-created team that hasn't yet been created on the remote DB